### DISCUSSION TITLE
Branches pattern with wildcard as prefix & suffix

### DISCUSSION THREAD

**Discussion by mochadwi** [community/community#26287] [Actions]
<pre><code class="lang-auto">on:
  pull_request:
    branches:    
      - '**prod**'
</code></pre>
<p>Using the above code seems not possible for us, does it currently not supported?</p>

**Comment by mochadwi** [comment, marked as answer]
<p>Turns out, the targetted branch is not a default branch, and no workflows added (exist previously) in the targetted branch, making it unable to trigger any <code>pull_request</code> event</p>

**Comment by Akash1134** [comment]
<p>Thanks for sharing the outcome <a class="mention" href="/mochadwi">@mochadwi</a> as it will help other users who might have the same question.</p>
